mage_ai/data_preparation/templates/data_loaders/s3.py

```diff
@@ -0,0 +1,16 @@
+from mage_ai.data_loader.s3 import S3
+from pandas import DataFrame
+
+
+def load_from_s3_bucket() -> DataFrame:
+    """
+    Template code for loading data from S3 bucket.
+
+    This template assumes that user credentials are specified in `~/.aws`.
+    If not, use `S3.with_credentials()` to manually specify AWS credentials or use
+    AWS CLI to configure credentials on system.
+    """
+    bucket_name = 'your_s3_bucket_name'  # Specify S3 bucket name to pull data from
+    object_name = 'your_object_name'  # Specify object to download from S3 bucket
```

Comment:
You can call it object_key, which includes the path to the object.

Reply:
Refactored code to use `object_key` instead of `object_name` (which is strange since AWS officially calls the identifiers `object_key` but `boto3` instead uses `object_name` in some places).